Fix code formatting style in tevent explanation

justin-stephenson opened the pull request:
Tevent code was not formatted into code blocks, making it difficult to interpret. I fixed this formatting and removed the `-` prefix which seemed unnecessary to me.

Also, the link to **internals_tevent.jpg** is not working, it seems this file is no longer in the repo.

justin-stephenson committed:
Fix code formatting style in tevent explanation

Remove tevent image that does not exist.

pbrezina commented:
I'm not sure if `~~~` will work since github is using ```. Can you please change that? Or did you check that it works?

The file is not even present in the original repo on pagure, so I'm afraid we can not recover it.

pbrezina commented:
Even `git log --all --full-history -- **/internals_tevent.jpg` does not find it so it was probably never commited.

justin-stephenson commented:
> I'm not sure if `~~~` will work since github is using ```. Can you please change that? Or did you check that it works?
> 
I just changed it to use the \``` c and \``` markdown formatting, matching what I found elsewhere.

> The file is not even present in the original repo on pagure, so I'm afraid we can not recover it.

I removed the text line mentioning this image.

Hope this helps.

justin-stephenson commented:
I tested this locally with `bundle exec jekyll serve` and it looks good to me.
